src/main/java/org/mastodon/views/trackscheme/display/style/TrackSchemeStyle.java

```diff
@@ -372,6 +374,11 @@ public boolean isHierarchyGraphCurvedLines()
 		return hierarchyCurvedLines;
 	}
 
+	public boolean isFadeFutureTimepoints()
+	{
+		return fadeFutureTimepoints;
+	}
+
```

Comment:
Having this setting in the TrackSchemeStyle feels wrong to me. It is too much hidden from the user. And it also feels like a misuse of the TrackSchemeStyle. I think now with the FadedModel it is possible to have a icon/button in the MamutBranchViewTrackSchemeHierarchy window, that enables or disables the FadedModel accordingly.